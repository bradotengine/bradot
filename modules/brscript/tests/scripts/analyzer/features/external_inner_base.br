extends "inner_base.br".InnerA.InnerAB

func test():
	super.test()
