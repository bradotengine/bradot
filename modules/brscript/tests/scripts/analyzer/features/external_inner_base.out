BRTEST_OK
InnerA.InnerAB.test
InnerB.test
